Duplicati/Library/AutoUpdater/UpdaterManager.cs

```diff
@@ -1075,7 +1075,7 @@ public static int RunFromMostRecent(System.Reflection.MethodInfo method, string[
         public static int RunFromMostRecentSpawn(System.Reflection.MethodInfo method, string[] cmdargs, AutoUpdateStrategy defaultstrategy = AutoUpdateStrategy.CheckDuring)
         {
             // If the update is disabled, go straight in
-            if (DISABLE_UPDATE_DOMAIN)
+            if (!DISABLE_UPDATE_DOMAIN)

```

Comment:
I have a setting disabling the updates via an environment variable in the `.csproj`: 

`AUTOUPDATER_Duplicati_SKIP_UPDATE=1`

Reply:
:) My hacky way for debugging. Thanks for the tip.